src/pages/RepoPage/CoverageOnboarding/CircleCI/CircleCIOrgToken.tsx

```diff
@@ -28,12 +28,15 @@ function CircleCIOrgToken() {
   const { data } = useRepo({ provider, owner, repo })
   const { data: orgUploadToken } = useOrgUploadToken({ provider, owner })
 
+  const uploadToken = orgUploadToken ?? data?.repository?.uploadToken
```

Comment:
Yeah, so this would be very useful if we are inclined to use a token over an other in most cases. Idk if there's an explicit hierarchy like "if there's a global token, use that, otherwise, if there's a repo token, use that, otherwise, if there's a github app integration, use that token instead, otherwise use tokenless, otherwise ...." you know? Idk if this applies/is correct, but if it was, it would be super helpful as we'd centralize a fn to define which token an org should use over another 🤷

Reply:
would defiantly be helpful if we have these switch cases!